daemon/list.go

```diff
@@ -134,6 +134,7 @@ func (daemon *Daemon) Containers(config *ContainersConfig) ([]*types.Container,
 			Names: names[container.ID],
 		}
 		newC.Image = container.Config.Image
+		newC.HostConfig.ContainerHostConfig = string(container.HostConfig().NetworkMode)
```

Comment:
Why call it `ContainerHostConfig` instead of `NetworkMode`? It can be other things than `"host"`.